src/Dependencies/Updater.php

```diff
@@ -87,12 +95,25 @@ private function update(array $dependencies)
 		return \Result\Result::ok();
 	}
 
-	private function prepareForUpdate()
+	private function prepareForUpdate($prepareForLanguageUpdate = false)
 	{
 		$result = $this->loadWhippetFiles();
 		if ($result->isErr()) {
 			return $result;
 		}
+		if ($prepareForLanguageUpdate && !empty($this->jsonFile->getDependencies(DependencyTypes::LANGUAGES))) {
+			$describer = new Describer($this->factory, $this->dir);
+			// FIXME: If whippet.lock does not exist, we cannot run the describer,
+			// and so we cannot get plugin and version information and we cannot
+			// update languages.
+			$result = $describer->fetchVersionInformation();
```

Comment:
This needs a further refactor of the Describer class, so that it doesn't always have to read from disk if we haven't yet written out the lockfile